browser-test/login.test.js

```diff
@@ -0,0 +1,63 @@
+import {Builder, By, until} from "selenium-webdriver";
+import {getDriver, getElementByXPath, urls, initialData} from "./utils";
+
+const firefox = require("selenium-webdriver/firefox");
+
+describe("Selenium tests for <Login />", () => {
+  let driver;
+  beforeAll(async () => {
+    jest.setTimeout(30000);
+    driver = await getDriver(Builder, new firefox.Options().headless());
```

Comment:
can you find a way to call the initialization python script from here?
I am not sure if the virtualenv would be used if activated, you could try.